### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by wildernessfamily** [pierre-alain-b/rainloop-nextcloud#279]
Hi,
First I wanna say thank you for all your time and hard work you placed into building this extension for NextCloud. Rainloop is amazing! I started off with the featured Mail extension with NextCloud and it's just not feasible to use in a working environment. For one, you need the basic feature of an archive button.

Rainloop is super easy to set up and it's so easy to use. Thank you! 

I saw version 7.2.5 of the Rainloop NextCloud extension. I looked for a Changelog but couldn't find one. I always like to read the changelog so I know what to look out for during an update in case something doesn't always update correctly and or if there is a security update on what the security update is fixing so I can check the logs for any type of previous intrusion and or feature update if a family member or employee asks what ??? this is :).

I saw that there was an old issue that was created back in 2017 asking the same question. The answer was that the updates correspond with the normal RainLoop app. But I'm confused. Right now RainLoop is 1.16.0 at https://www.rainloop.net/changelog/ 
However, in NextCloud the update is 7.2.5. When checking https://www.rainloop.net/changelog/ I can't find any dates to know when the releases come out to correspond with the NextCloud RainLoop updates. 
Is there a changelog for the NextCloud extension with dates to reference?

Thank you,

**Comment by daffydock** [comment]
Hi, 

Meanwhile you wait for a response from the Dev, I will try to help out a bit and let you know that Rainloop and NC-Rainloop are not on par on release versions, clearly. 

Further, albeit the Rainloop app has not have had any real discernible updates in quite a while, the NC App has, if only to keep it working with Nextcloud, as they do updates onto the server. 

For your reference, updates/changelogs of the NC-Rainloop app can usually be found along with the releases here on Github.
https://github.com/pierre-alain-b/rainloop-nextcloud/releases 

Cheers.

**Comment by pierre-alain-b** [comment]
Well indeed this is not well documented and I understand the number of rainloop-nextcloud is not aligned with the one from Rainloop. Still you have some clues in https://github.com/pierre-alain-b/rainloop-nextcloud/releases.

The reason for this is that the version was already 4.x when I ported the rainloop owncloud app to nextcloud. I've continued numbering from there although this is unrelated to Rainloop version numbers.

I will try to be more precise in the releases description in the future.
